content/review.rst

```diff
@@ -201,9 +201,24 @@ A merge commit should have a commit message, in the format::
   commit to a ticket / branch / review / task and check the history/story of
   that commit.
 
+A note on Trac review requests
+==============================
+
+* When submitting a Trac ticket for review, please bear in mind that this is
+  the moment to add relevant documentation you updated, affected repos, etc.
+  
+* A review in one of your Trac tickets is not just to say "please look through
+  this issue to see that I worked so I can close it", but instead a "I have
+  done this, documented it here, and pushed this PR. Please take a look and
+  let me what you think". This is clearer on Github as there is always code
+  and changes already documented on the PRs themselves. 
```

Comment:
it might be clearer in GitHub, but also for github, if there are external changes, the should be described in the ticket.

review request done in Trac should follow the same template as the one used on GitHub.

so the idea is that the review template should be used for any review request, Trac, GitHub, Email ...